src/modules/freia/FreiaBase.cpp

```diff
@@ -69,8 +70,7 @@ FreiaBase::FreiaBase(BaseSettings const &settings, struct FreiaSettings &LocalFr
   // VMM3Parser stats
   Stats.create("readouts.error_size", Counters.VMMStats.ErrorSize);
   Stats.create("readouts.error_ring", Counters.VMMStats.ErrorRing);
-  Stats.create("readouts.ring_mismatch", Counters.RingCfgErrors);
-  Stats.create("readouts.fen_mismatch", Counters.FENCfgErrors);
```

Comment:
Add Stats.create for Counters.RingErrors and FENErrors, and check if they're being used

Reply:
Not being used, deleted from counters, Ring and FEN errors caught in VMMStats